backend/app/main.py

```
@@ -0,0 +1,52 @@
+@app.get("/", tags=["Root"])
+async def read_root():
+    return {"message": f"Welcome to {settings.PROJECT_TITLE}!"}
+
+app.include_router(auth_router, prefix="/api/v1/auth", tags=["Auth"])
+#other routes will go here    
+
+# Run the application
+if __name__ == "__main__":
+    uvicorn.run(
+        "main:app",
+        host="0.0.0.0",
```

Ruff (0.8.2)

Possible binding to all interfaces

(S104)